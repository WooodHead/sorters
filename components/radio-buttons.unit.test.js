import RadioButtons from './radio-buttons'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('radio-buttons', () => {
    it('shallow-renders', () => {
        const component = shallow(<RadioButtons
            id="buttons"
            label="Buttons"
            defaultValue="foo"
            values={{
                foo: {
                    label: 'Foo',
                },
                bar: {
                    label: 'Bar'
                },
            }}
        />)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
    })
})
