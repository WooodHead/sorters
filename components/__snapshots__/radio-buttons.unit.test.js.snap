// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`radio-buttons shallow-renders 1`] = `
<div
  className="form-group"
>
  <label>
    Buttons
  </label>
  <div
    className="radio"
  >
    <label>
      <input
        defaultChecked={true}
        id="buttons"
        name="buttons"
        type="radio"
        value="foo"
      />
       
      Foo
    </label>
  </div>
  <div
    className="radio"
  >
    <label>
      <input
        defaultChecked={false}
        id="buttons"
        name="buttons"
        type="radio"
        value="bar"
      />
       
      Bar
    </label>
  </div>
</div>
`;
